src/sections/Resources/Resources-grid/paginate.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ResourcePageWrapper } from './resourceGrid.style';
+
+const Pagination = ({ postsPerPage, totalPosts, paginate, currentPage }) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(totalPosts / postsPerPage); i++) {
+    pageNumbers.push(i);
+  }
+
+  return (
+      <ResourcePageWrapper>
```

Comment:
No need to import this stylesheet and add another div for just adding the margins.
It could be done by simply adding the margins to the div below it.